tests/css-borders-4.js

```diff
@@ -149,49 +186,438 @@ export default {
-				'round angle round angle 2px',
-				'2px round',
-				'2px 4px round',
-				'2px 4px 8px round',
-				'2px 4px 8px 16px round',
-				'round 2px / 2px',
-				'round 2px / 2px 4px',
-				'round 2px / 2px 4px 8px',
-				'round 2px / 2px 4px 8px 16px',
+				'10%',
+				'1fr',
+				'10px 20px',
+				'10% 20%',
+				'1fr 2fr',
+				'10px 20% 1fr',
+			],
+		},
+		'border-clip-top': {
+			links: {
+				dev: '#border-clip',
+			},
+			tests: [
+				'normal',
+				'10px',
+				'10%',
+				'1fr',
+				'10px 20px',
+				'10% 20%',
+				'1fr 2fr',
+				'10px 20% 1fr',
+			],
+		},
+		'border-clip-right': {
+			links: {
+				dev: '#border-clip',
+			},
+			tests: [
+				'normal',
+				'10px',
+				'10%',
+				'1fr',
+				'10px 20px',
+				'10% 20%',
+				'1fr 2fr',
+				'10px 20% 1fr',
+			],
+		},
+		'border-clip-bottom': {
+			links: {
+				dev: '#border-clip',
+			},
+			tests: [
+				'normal',
+				'10px',
+				'10%',
+				'1fr',
+				'10px 20px',
+				'10% 20%',
+				'1fr 2fr',
+				'10px 20% 1fr',
+			],
+		},
+		'border-clip-left': {
+			links: {
+				dev: '#border-clip',
+			},
+			tests: [
+				'normal',
+				'10px',
+				'10%',
+				'1fr',
+				'10px 20px',
+				'10% 20%',
+				'1fr 2fr',
+				'10px 20% 1fr',
+			],
+		},
+		'box-shadow-color': {
+			links: {
+				dev: '#box-shadow-color',
+			},
+			tests: [
+				'green',
+				'green, blue'
+			],
+		},
+		'box-shadow-offset': {
+			links: {
+				dev: '#box-shadow-offset',
+			},
+			tests: [
+				'none',
+				'0 0',
+				'10px 1em',
+				'-10px -1em',
+				'none, 0 0, 10px 1em',
+			],
+		},
+		'box-shadow-blur': {
+			links: {
+				dev: '#box-shadow-blur',
+			},
+			tests: [
+				'0',
+				'10px',
+				'1em',
+				'10px, 1em',
+			],
+		},
+		'box-shadow-spread': {
+			links: {
+				dev: '#box-shadow-spread',
+			},
+			tests: [
+				'0',
+				'10px',
+				'1em',
+				'10px, 1em',
+			],
+		},
+		'box-shadow-position': {
+			links: {
+				dev: '#box-shadow-position',
+			},
+			tests: [
+				'outset',
+				'inset',
+				'outset, inset',
+			],
+		},
+		'border-shape': {
+			links: {
+				dev: '#border-shape',
+			},
+			tests: [
+				'none',
+				'inset(10% round 10% 40% 10% 40%)',
+				'ellipse(at top 50% left 20%)',
+				'circle(at top left)',
+				'polygon(100% 0, 100% 100%, 0 100%)',
+				"path('M 20 20 H 80 V 30')",
+				'rect(10% 20px 30% 40px)',
+				'xywh(10% 40% 100px 200px round 10% 40% 10% 40%)',
+				'inset(10% round 10% 40% 10% 40%) margin-box',
+				'ellipse(at top 50% left 20%) margin-box',
+				'circle(at top left) margin-box',
+				'polygon(100% 0, 100% 100%, 0 100%) margin-box',
+				"path('M 20 20 H 80 V 30') margin-box",
+				'rect(10% 20px 30% 40px) margin-box',
+				'xywh(10% 40% 100px 200px round 10% 40% 10% 40%) margin-box',
+				'attr(src url)',
+				'url(image.png)',
+				'margin-box',
+				'border-box',
+				'padding-box',
+				'content-box',
+				'fill-box',
+				'stroke-box',
+				'view-box',
+				'inset(10% round 10% 40% 10% 40%) margin-box',
```

Comment:
There is already inset() in the second item of the array, can you move this together?

Reply:
That test was actually redundant. So I removed it.